book/06-github/sections/1-setting-up-account.asc

```diff
@@ -1,95 +1,95 @@
-=== Account Setup and Configuration
+=== Pag-setup at pagsasaayos ng Account
 
 (((GitHub, user accounts)))
-The first thing you need to do is set up a free user account.
-Simply visit https://github.com[], choose a user name that isn't already taken, provide an email address and a password, and click the big green ``Sign up for GitHub'' button.
+Ang unang bagay na kailangan mong gawin ay mag-setup ng isang libreng account ng user.
+Bisitahin lamang ang https://github.com[], pumili ng pangalan ng user na hindi pa  nakuha, magbigay ng isang email address at password, at i-click ang malaking berdeng pindutan na ``Mag-sign up sa GitHub''.
 
-.The GitHub sign-up form.
+.Ang form ng pag-sign-up sa GitHub.
 image::images/signup.png[The GitHub sign-up form.]
 
-The next thing you'll see is the pricing page for upgraded plans, but it's safe to ignore this for now.
-GitHub will send you an email to verify the address you provided.
-Go ahead and do this, it's pretty important (as we'll see later).
+Ang susunod na bagay na iyong makikita ay ang pahina ng pagpepresyo para sa naka-upgrade na mga plano, ngunit ito'y ligtas na huwag pansinin ito sa ngayon.
+Magpapadala sa iyo ng email ang GitHub upang mapatunayan ang address na iyong ibinigay.
+Sige at gawin ito, ito ay mahalaga (tulad ng makikita natin kinamamayaan).
 
-[NOTE]
+[TANDAAN]
 ====
-GitHub provides all of its functionality with free accounts, with the limitation that all of your projects are fully public (everyone has read access).
-GitHub's paid plans include a set number of private projects, but we won't be covering those in this book.
+Ang GitHub ay nagbibigay ng lahat ng pag-andar nito na may mga libreng account, may limitasyon na ang lahat ng iyong mga proyekto ay ganap na pampubliko (bawat isa ay may access sa pagbasa).
+Kasama sa mga bayad na plano ay isang hanay ng bilang ng mga pribadong proyekto, ngunit hindi natin tatalakayin sa librong ito.
 ====
 
-Clicking the Octocat logo at the top-left of the screen will take you to your dashboard page.
-You're now ready to use GitHub.
+Pag-clikc sa Ocotocat na logo sa itaas sa kaliwa ng screen ay magdadala sa iyo sa pahina ng iyong dashboard.
```

Comment:
Pag-clikc  changed to Pag-click

Reply:
hi sir @robin-ho, i have updated my PR. thank you for pointing out my typos :)